### ISSUE TITLE
Measure tools silently fail for 1D objects

### ISSUE THREAD

**Issue by MatthiasJ1** [bernhard-42/vscode-ocp-cad-viewer#93]
```py
from ocp_vscode import *
from build123d import *

x = Line(0, 1)

show(x, measure_tools=True)
```
Works as expected with `measure_tools=False`.

**Comment by bernhard-42** [comment]
Already works in my latest version, great!

**Comment by bernhard-42** [comment]
Fixed with 2.4.0
